makelib/docs.mk

```diff
@@ -42,6 +42,7 @@ docs.init:
 	rm -rf $(DOCS_WORK_DIR)
 	mkdir -p $(DOCS_WORK_DIR)
 	git clone --depth=1 -b master $(DOCS_GIT_REPO) $(DOCS_WORK_DIR)
+	(cd $(DOCS_WORK_DIR) && npm install)
```

Comment:
This may not be required in all cases -- would suggest adding as a dependency to targets in https://github.com/crossplane/docs/blob/9fd142c5c855630dd8a55a4a19194428243a42b6/Makefile#L76

Reply:
I've added an `npm.install` make target to the docs in https://github.com/crossplane/docs/pull/233. And called `make npm.install` here instead.

It is needed here because `docs.init` prunes and recreates the docs repo so all installed are packages are lost, too.